*** jquery.bootstrap.wizard.js.old	2013-03-18 23:05:30.610197306 +0100
--- jquery.bootstrap.wizard.js	2013-03-18 23:06:20.978194434 +0100
***************
*** 168,174 ****
  	// Work the next/previous buttons
  	obj.fixNavigationButtons();
  
! 	$('a[data-toggle="tab"]', element).on('click', function (e) {
  	    // Get the index of the clicked tab
  	    var clickedIndex = $navigation.find('li').index($(e.currentTarget).parent('li'));
  	    if($settings.onTabClick && typeof $settings.onTabClick === 'function' && $settings.onTabClick($activeTab, $navigation, obj.currentIndex(), clickedIndex)===false){
--- 168,174 ----
  	// Work the next/previous buttons
  	obj.fixNavigationButtons();
  
! 	$('a[data-toggle="wtab"]', element).on('click', function (e) {
  	    // Get the index of the clicked tab
  	    var clickedIndex = $navigation.find('li').index($(e.currentTarget).parent('li'));
  	    if($settings.onTabClick && typeof $settings.onTabClick === 'function' && $settings.onTabClick($activeTab, $navigation, obj.currentIndex(), clickedIndex)===false){
***************
*** 176,182 ****
  	    }
  	});
  
! 	$('a[data-toggle="tab"]', element).on('show', function (e) {
  	    $element = $(e.target).parent();
  	    // If it's disabled then do not change
  	    if($element.hasClass('disabled')) {
--- 176,182 ----
  	    }
  	});
  
! 	$('a[data-toggle="wtab"]', element).on('show', function (e) {
  	    $element = $(e.target).parent();
  	    // If it's disabled then do not change
  	    if($element.hasClass('disabled')) {
